libr/core/visual.c

```diff
@@ -769,25 +769,28 @@ static void findPrevWord(RCore *core) {
 
 // TODO: integrate in '/' command with search.inblock ?
 static void visual_search(RCore *core) {
+#define BUF_SZ 256
 	const ut8 *p;
 	int len, d = core->print->cur;
-	char str[128], buf[258];
+	char str[128], buf[BUF_SZ];
```

Comment:
dont use defines to assume the size of a buffer, this may lead to inconsistencies if not refactored properly. it is recommended to u se sizeof()